docs/guides/provider-configuration.md

```diff
@@ -36,6 +36,22 @@ provider "mongodbatlas" {
 
 **Note:** SAs can't be used with `mongodbatlas_event_trigger` resources because its API doesn't support it yet.
 
+**Troubleshooting Service Accounts**
+
+If you encounter a rate limit error when using Service Accounts, you may see:
```

Comment:
`may` describes permission to do something. `might` describes the potential for something to happen
```suggestion
If you encounter a rate limit error when using Service Accounts, you might see:
```